code/modules/error_handler/error_handler.dm

```diff
@@ -96,7 +96,7 @@ var/total_runtimes_skipped = 0
 
 	// Now to actually output the error info...
 	log_world("\[[time_stamp()]] Runtime in [e.file],[e.line]: [e]")
-	log_runtime_txt("\[[time_stamp()]] Runtime in [e.file],[e.line]: [e]")
+	log_runtime_txt("Runtime in [e.file],[e.line]: [e]")
```

Comment:
Uhh?

Reply:
rust-g automatically adds timestamps now, doing this results in duplicated timestamps.

Reply:
right-o.